openmc/mesh.py

```diff
@@ -1049,6 +1049,8 @@ class CylindricalMesh(StructuredMesh):
         The default value is [0, 2π], i.e. the full phi range.
     z_grid : numpy.ndarray
         1-D array of mesh boundary points along the z-axis.
+    origin : tuple
```

Comment:
We'll want to update the XML spec as well https://github.com/openmc-dev/openmc/blob/develop/docs/source/io_formats/tallies.rst#mesh-element

Reply:
Done